state-sync/diem-data-client/src/diemnet/state.rs

```diff
@@ -118,14 +122,34 @@ impl PeerStates {
     }
 
     pub fn update_score_success(&mut self, peer: PeerNetworkId) {
-        self.inner.entry(peer).or_default().update_score_success()
+        let old_score = self.inner.entry(peer).or_default().score;
+        self.inner.entry(peer).or_default().update_score_success();
+        let new_score = self.inner.entry(peer).or_default().score;
+        if old_score <= IGNORE_PEER_THRESHOLD && new_score > IGNORE_PEER_THRESHOLD {
```

Comment:
I could have gone with just logging the score whenever it is updated or as long as it's above/below the threshold. However, it seems that we are more interested in the events when a peer switches between ignored/no longer ignored. Please let me know if we just want to log the score and I can change.